### PR TITLE
Added kitchen and food related classes

### PR DESCRIPTION
**Pull request by samuelpg**
* Added `KitchenUtensil` class with subclasses.
* Added `FoodOrDrink` class with subclasses.
* Added class `Drawer` to parent class `Furniture`
* Added more appliances sub-classes to `Appliance`

### CODE REVIEW

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]
* Can you please make sure that the class definitions in `apartment.owl` and `homelab.owl` are in sync?
* Please delete `test.py`

**Comment by samuelpg** [comment]
* Made `apartment.owl` consistent with `homelab.owl`
* deleted `test.py`

File: common/ontology/apartment.owl [hunk 1]
```diff
@@ -1,441 +1,1217 @@
...
+        <rdfs:domain rdf:resource="apartment:Object"/>
+        <rdfs:range rdf:resource="apartment:Object"/>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- apartment:below -->
+
+    <owl:ObjectProperty rdf:about="apartment:below">
+        <owl:inverseOf rdf:resource="file:/home/samuel/Documents/MAS/b-it-bots/mas_knowledge_base/common/ontology/apartment.owl#apartment:above"/>
```
Comment (alex-mitrevski) [inline common/ontology/apartment.owl]: Can you please remove these local paths (here and everywhere below)?

**Review by alex-mitrevski — CHANGES_REQUESTED** [review]

**Comment by samuelpg** [comment]
* Removed local paths in Object Properties

**Review by alex-mitrevski — APPROVED** [review]

**Pull request merged**